scripts/irods/test/test_iquery.py

```diff
@@ -106,3 +106,13 @@ def test_iquery_supports_embedded_single_quotes__issue_5727(self):
             query_string = f"select DATA_NAME, COLL_NAME where COLL_NAME = '{self.user.session_collection}' or DATA_NAME = '{escaped}'"
             with self.subTest(f'escaped_string={escaped}'):
                 self.user.assert_icommand(['iquery', query_string], 'STDOUT', [json_string])
+
+    def test_iquery_returns_error_on_invalid_query_string__issue_5734(self):
+        query_string = "select DATA_ID, DATA_REPL_NUM, COLL_NAME, DATA_NAME, DATA_RESC_HIER, DATA_PATH, META_DATA_ATTR_VALUE " + \
+            "where META_DATA_ATTR_NAME = 'id_foo and META_DATA_ATTR_VALUE in ('260,'3261852','3261856','3261901','3080907','3083125'," + \
```

Comment:
For what it's worth...

This appears to be similar/the same problem as #6393 (the problem is that there is a missing single quote after `'260`). Should we combine these tests OR use the other example test from https://github.com/irods/irods/issues/5734#issuecomment-876397457?

Reply:
done